From 166f17ab62e112a4fdfd3afd2956844b54f05aab Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 11 Oct 2019 18:49:50 +0800
Subject: [PATCH] arm64: rockchip_defconfig: remove unnessesary errata config

-CONFIG_CAVIUM_ERRATUM_30115
-CONFIG_QCOM_FALKOR_ERRATUM_1003
-CONFIG_QCOM_FALKOR_ERRATUM_1009
-CONFIG_QCOM_QDF2400_ERRATUM_0065
-CONFIG_SOCIONEXT_SYNQUACER_PREITS
-CONFIG_HISILICON_ERRATUM_161600802
-CONFIG_QCOM_FALKOR_ERRATUM_E1041

Change-Id: Iba5bf9d37ebd2978deff40716091b820ee8d7b5e
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c277589335f2..f0b6b8121617 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -52,6 +52,13 @@ CONFIG_PCIE_ROCKCHIP_HOST=y
 # CONFIG_CAVIUM_ERRATUM_22375 is not set
 # CONFIG_CAVIUM_ERRATUM_23154 is not set
 # CONFIG_CAVIUM_ERRATUM_27456 is not set
+# CONFIG_CAVIUM_ERRATUM_30115 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_1003 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_1009 is not set
+# CONFIG_QCOM_QDF2400_ERRATUM_0065 is not set
+# CONFIG_SOCIONEXT_SYNQUACER_PREITS is not set
+# CONFIG_HISILICON_ERRATUM_161600802 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_E1041 is not set
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=8
 CONFIG_HZ_300=y
-- 
2.35.3

